atlp-rwanda/e-comm-team-techsmith-fn
# #185218915 Users should be able to Login via Google.

## What does this PR do?
This PR allows Users sign into a Google account, which helps you to quickly and easily manage user authentication on your website.
## Description of Task to be completed?
-  Provide their consent, and securely share their profile information with your platform. 
-  Customizable buttons and multiple flows are supported for user sign-up and sign-in.
-  Adding Pull Request template adhering to ATLP's Engineering Playbook

## How should this be manually tested?

- Clone the repository using `git clone https://github.com/atlp-rwanda/e-comm-team-techsmith-fn.git` 

- Checkout to the branch using `git checkout ft-users-should-be-able-to-login-via-google-185218915`

- Install node packages that the application depends on: `npm i`

- Create a `.env` file in the root directory and add all environment variables required to run the application. The required variables are listed in the `.env.example` file available in root directory of the application.

- Start the application development server: `npm start`

## What are the relevant pivotal tracker/Trello stories?
+[185218915](https://www.pivotaltracker.com/story/show/185218915)

## Screenshots (if appropriate)
<img width="1263" alt="Screenshot 2023-06-06 at 20 09 38" src="https://github.com/atlp-rwanda/e-comm-team-techsmith-fn/assets/64655121/52cf9316-d399-4e72-95c3-699c90b8e267">
